### PR TITLE
Ft/day 5

### PR DESCRIPTION
**Pull request by userdtuan**
add bootstrap and deloy to heroku

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,6 +11,21 @@
   </head>
 
   <body>
-    <%= yield %>
+  <div class="container">
```
Comment (ttduongtran) [inline app/views/layouts/application.html.erb:14]: Line 15 - 25  will be better if you use a partial

**Comment by ttduongtran** [comment]
Pls put node_module into file `.gitignore`. You should not push node_module to git.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (userdtuan) [reply app/views/layouts/application.html.erb:14]: oke sis, I will practice using partial

**Comment by userdtuan** [comment]
> Pls put node_module into file `.gitignore`. You should not push node_module to git.

I will fix this!

**Pull request merged**